web/src/main/java/it/pagopa/selfcare/pagopa/backoffice/web/model/mapper/GecMapper.java

```diff
@@ -0,0 +1,98 @@
+package it.pagopa.selfcare.pagopa.backoffice.web.model.mapper;
+
+import it.pagopa.selfcare.pagopa.backoffice.connector.model.gec.Bundle;
+import it.pagopa.selfcare.pagopa.backoffice.connector.model.gec.Bundles;
+import it.pagopa.selfcare.pagopa.backoffice.connector.model.gec.Touchpoint;
+import it.pagopa.selfcare.pagopa.backoffice.connector.model.gec.Touchpoints;
+import it.pagopa.selfcare.pagopa.backoffice.web.model.gec.BundleResource;
+import it.pagopa.selfcare.pagopa.backoffice.web.model.gec.BundlesResource;
+import it.pagopa.selfcare.pagopa.backoffice.web.model.gec.TouchpointResource;
+import it.pagopa.selfcare.pagopa.backoffice.web.model.gec.TouchpointsResource;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class GecMapper {
+
+    public static BundleResource toResource(Bundle bundle){
+        if(bundle == null){
+            return null;
+        }
+        BundleResource response = new BundleResource();
```

Comment:
if the source object is identical to the destination object you can use [modelMapper](https://modelmapper.org/). It maps in automatic way two objects.

Reply:
ok thanks for the new option I will try to implement it in the next development